pyfolio/bayesian.py

```diff
@@ -86,7 +86,7 @@ def model_returns_t_alpha_beta(data, bmark, samples=2000):
                  mu=mu_reg,
                  sd=sigma,
                  observed=y)
-        trace = pm.sample(samples)
+        trace = pm.sample(samples, progressbar=False)
```

Comment:
This seemed to have fixed the issue but it's not a proper fix. Can you move this up to be a kwarg of the function, default to `True` and set it to `False` in the unittest?